it/cassandra/src/main/java/org/apache/beam/it/cassandra/CassandraResourceManagerUtils.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.it.cassandra;
+
+import static org.apache.beam.it.common.utils.ResourceManagerUtils.generateResourceId;
+
+import com.google.re2j.Pattern;
```

Comment:
Why this over the stuff that ships with Java? It seems basically the same

Reply:
Changed to use java Patterns in https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/710.